db/schema.rb

```diff
@@ -10,12 +10,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_01_25_162730) do
-
+ActiveRecord::Schema[7.0].define(version: 2023_01_17_125523) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
+  create_table "annotations", force: :cascade do |t|
+    t.bigint "medium_id", null: false
+    t.bigint "user_id", null: false
+    t.text "timestamp"
+    t.string "color"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.text "comment"
+    t.boolean "visible_for_teacher"
+    t.integer "category"
+    t.index ["medium_id"], name: "index_annotations_on_medium_id"
+    t.index ["user_id"], name: "index_annotations_on_user_id"
+  end
+
```

Comment:
Do we need to update the schema in order to update versions in this PR?

Reply:
No we should definitely not. This is an artefact from working in a different branch with the saame db in development.